frontend/pages/discover/projects.tsx

```diff
@@ -101,29 +105,28 @@ const ProjectsPage: PageComponent<ProjectsPageProps, DiscoverPageLayoutProps> =
   return (
     <>
       <Head title={intl.formatMessage({ defaultMessage: 'Discover Projects', id: 'Qt/+mk' })} />
-      <div className="relative flex flex-col w-full h-full lg:gap-0 lg:overflow-hidden lg:flex-row">
+      <div className="relative flex flex-col w-full h-full md:gap-0 md:overflow-hidden md:flex-row">
         <div
           ref={projectsListAndDetailsContainerRef}
-          className="flex flex-col lg:top-1 lg:bottom-1 lg:absolute lg:w-full"
+          className="flex flex-col md:top-1 md:bottom-1 md:absolute md:w-full"
         >
-          <div className="relative flex flex-col lg:overflow-hidden lg:w-5/12">
+          <div className="relative flex flex-col md:overflow-hidden md:w-5/12">
             <div
               ref={projectsListContainerRef}
               className={cx({
-                'relative flex-grow lg:pr-2.5': true,
-                'lg:overflow-y-auto': !loading,
-                'lg:pointer-events-none lg:overflow-hidden': loading,
+                'relative flex-grow md:pr-2.5': true,
+                'md:overflow-y-auto': !loading,
+                'md:pointer-events-none md:overflow-hidden': loading,
               })}
             >
               {loading && (
                 <span className="absolute bottom-0 z-20 flex items-center justify-center bg-gray-600 bg-opacity-20 top-1 left-1 right-3 rounded-2xl">
                   <Loading visible={loading} iconClassName="w-10 h-10" />
                 </span>
               )}
-              <div className="flex flex-col">
+              <div className="flex flex-col gap-2 py-0.5 md:pl-0.5">
```

Comment:
For consistency with all the other card types, let's set the same padding:
```suggestion
              <div className="flex flex-col gap-2 p-0.5">
```